includes/view/User_view.php

```diff
@@ -37,23 +37,22 @@ function User_settings_view(
     $enable_dect = config('enable_dect');
     $enable_planned_arrival = config('enable_planned_arrival');
 
-    $showOauth = false;
-    foreach ($oauth2_providers as $name => $config) {
-        $showOauth = $showOauth || !isset($config['hidden']) || !$config['hidden'];
-    }
-
     return page_with_title(settings_title(), [
         msg(),
         div('row', [
-            div('col-md-6', [
+            div('col-md-3 settings-menu', [
+                '<ul class="nav nav-pills nav-stacked">',
```

Comment:
That means that this navigation has to be changed in two places when editing it which is not that great, there has to be a better option using the twig template and adding the content directly to it

Reply:
Sorry, won't happen until settings are completely rewritten. Tried 5 things that didn't work.

Reply:
It works for me, see 35aff139f7c0d140cf1751b104fa2e5320ea6dba ;)